task/oci-copy-oci-ta/0.1/oci-copy-oci-ta.yaml

```diff
@@ -120,6 +122,7 @@ spec:
         IMAGE_INDEX_DIGEST=$(oras resolve --registry-config auth.json "${IMAGE}")
         echo -n "$IMAGE_INDEX_DIGEST" | tee "$(results.IMAGE_DIGEST.path)"
         echo -n "$IMAGE" | tee "$(results.IMAGE_URL.path)"
+        echo -n "${IMAGE}@${IMAGE_INDEX_DIGEST}" >"$(results.IMAGE_REF.path)"
```

Comment:
Could use tee for consistency I guess.

Reply:
Not sure why? The information is already on stdout, why display it the second time concatenated?

Reply:
Yeah, that's true.